src/runtime/test/manifest-test.ts

```diff
@@ -1982,15 +1982,19 @@ resource SomeName
       assert.lengthOf(manifest.particles, 1);
       const particle = manifest.particles[0];
       assert.isEmpty(particle.trustChecks);
-      assert.equal(particle.trustClaims.size, 2);
+      assert.equal(particle.trustClaims.length, 2);
```

Comment:
There is much better support for testing arrays than for maps, so you can use the lengthOf function:

assert.lengthOf(particle.trustClaims, 2);

Reply:
Done.